test/rsspp_parser.cpp

```diff
@@ -631,3 +631,14 @@ TEST_CASE("parse_url() assumes utf-8 if no encoding specified and replaces inval
 	REQUIRE(parsed_feed.items.size() == 1);
 	REQUIRE(parsed_feed.title == expected_title);
 }
+
+TEST_CASE("Exit with abort (fixtures)",
```

Comment:
I'd prefer a more descriptive name:

```suggestion
TEST_CASE("Throws if no \"channel\" element is found",
```